manifests/entandoapps.entando.org.crd.yaml

```diff
@@ -74,44 +58,33 @@ spec:
                   type: string
                 ingressHostName:
                   description: >
-                    The hostname that will be used to access this EntandoApp  from outside the cluster. The Entando
+                    The hostname that will be used to access this EntandoApp from outside the cluster. The Entando
                     Operator will create an Ingress reflecting this hostname, and expose this EntandoApp on the
                     path specified in 'EntandoApp.spec.ingressPath' on
-                    this host. If omitted, the Entando Operator will automatically generate a hostname using the value of
-                    the ENTANDO_DEFAULT_ROUTING_SUFFIX environment variable that the Entando Operator was configured with.
-                    It will prefix the name and namespace of the EntandoApp to this default routing suffix.
+                    this host. If omitted, the Entando Operator will automatically generate a hostname.
+
+                    See [here] (https://dev.entando.org/v6.3/docs/concepts/operator-intro.html#ingress-path-generation) for how the operator will create and configure the Ingress.
                   type: string
                   pattern: '^([A-Za-z0-9-]{1,63}\.)*[[A-Za-z0-9-]{1,63}$'
                 ingressPath:
                   description: >
                     The path that will be used to expose this EntandoApp on the ingress host
                     specified in 'EntandoApp.spec.ingressHostname'. This property needs to be in sync with
-                    the web context path that is used in the underlying WAR deployment. It should there either be
+                    the web context path that is used in the underlying WAR deployment. It should either be
```

Comment:
Do we document the WAR deployment setup somewhere? Not finding it in docs and  I wonder if we need something around controlling/changing the web context path itself. I can't recall if we had a ticket for that before we lost Jira but I know I've heard mention of needing to do that, switching out from entando-de-app and/or removing it completely. Just asking here if we need to get that in queue.